PixaPicker/PixaPicker.xcdatamodeld/PixaPicker.xcdatamodel/contents

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<model type="com.apple.IDECoreDataModeler.DataModel" documentVersion="1.0" lastSavedToolsVersion="14492.1" systemVersion="18G87" minimumToolsVersion="Automatic" sourceLanguage="Swift" userDefinedModelVersionIdentifier="">
+    <entity name="URL" representedClassName="URL" syncable="YES" codeGenerationType="class">
```

Comment:
When you create a coredata model called `URL`, it creates a hidden class called `URL`. This creates a naming collision since a class named `URL` already exists.  

To get the project running, I simply changed `URL` entity to `PhotoURL`